source/index.html

```diff
@@ -64,8 +64,40 @@ <h2 class="working-basics__title">Как это работает</h2>
       </li>
     </ul>
   </section>
+  <article class="demonstrating-example">
+    <h2 class="demonstrating-example__title">Живой пример</h2>
+    <p class="demonstrating-example__promo-text">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+    <ul class="demonstrating-example__infographics infographics">
```

Comment:
```suggestion
    <ul class="demonstrating-example__infographic infographic">
```
тут единственное число, тк это одна инфографика